source/manual/taxonomy.html.md

```diff
@@ -73,6 +73,20 @@ tagged to the above mentioned ["Education taxon"][education-taxon]:
 
 [https://www.gov.uk/api/search.json?filter_taxons[]=c58fdadd-7743-46d6-9629-90bb3ccc4ef0](https://www.gov.uk/api/search.json?filter_taxons[]=c58fdadd-7743-46d6-9629-90bb3ccc4ef0)
 
+By default rummager returns a handful of fields in a search result item.
+You are able to override the default fields by naming which fields you want returned. 
+If a content item does not have one of the named fields provided,
+it will be left out of the returned item.
+There is more info[here][override-fields] 
```

Comment:
I'd say something like `[View the full documentation here][override-fields].` as that is a more complete call to action (a anchor with "here" as the descriptive text is a bit lacking). Also, this fixes the spacing issue between "info" and "here", as well as the missing full stop in the above sentence.